modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+
+You can replace the default `infrastructureName` prefix, `mycluster-7977k`, in the log group names by specifying a `groupPrefix` field in the `ClusterLogForwarding` CR:
+
+[source,yaml]
+----
+cloudwatch:
+    region: us-east-2
+    groupBy: logType
+    groupPrefix: demo-group-prefix
+----
+
+The value of `groupPrefix` replaces the default `infrastructureName` prefix:
+
+[source,terminal]
+----
+$ aws --output json logs describe-log-groups | jq .logGroups[].logGroupName
+"demo-group-prefix.application"
+"demo-group-prefix.audit"
+"demo-group-prefix.infrastructure"
+----
+
+.Example: Naming log groups for application namespace names
+
+For each application namespace, you can create a log group whose name is based on the name of the application namespace.
+
+If you destroy a namespace object and create a new one that has the same application namespace name, CloudWatch continues using the same log group as before.
+
+If you consider successive objects with the same application namespace name as equivalent to each other, use the approach described here. Otherwise, if you need to distinguish these log groups from each other, see the following "Naming log groups for application namespace UUIDs" section instead.
+
+To create application log groups whose names are based on the names of the application namespaces, you set the value of the `groupBy` field to `namespaceName` in the `ClusterLogForwarder` CR:
+
+[source,terminal]
+----
+cloudwatch:
+    region: us-east-2
+    groupBy: namespaceName
+----
+
+The `groupBy` field affects the application log group only. It does not affect the `audit` and `infrastructure` log groups.
+
+In AWS, the namespace name appears at the end of each log group name. Because there is a single application namespace, "app", the following output shows a new `mycluster-7977k.app` log group instead of `mycluster-7977k.application`:
+
+[source,terminal]
+----
+$ aws --output json logs describe-log-groups | jq .logGroups[].logGroupName
+"mycluster-7977k.app"
+"mycluster-7977k.audit"
+"mycluster-7977k.infrastructure"
+----
+
+If the cluster in this example had contained multiple application namespaces, the output would show multiple log groups, one for each namespace.
+
+The `groupBy` field affects the application log group only. It does not affect the `audit` and `infrastructure` log groups.
+
+.Example: Naming log groups for application namespace UUIDs
+
+For each application namespace, you can create a log group whose name is based on the UUID of the application namespace.
+
+If you destroy a namespace object and create a new one, CloudWatch creates a new log group.
+
+If you consider successive objects with the same application namespace name as different from each other, use the approach described here. Otherwise, see the preceding "Example: Naming log groups for application namespace names" section instead.
+
+To do this, you set the value of the `groupBy` field to `namespaceUUID` in the `ClusterLogForwarder` CR:
```

Comment:
`this` is vague. Simply say what you mean. Reading the preceding paragraph, I think you mean, 

`To distinguish the same application namespaces from one each other, set the value of the groupBy filed to...`